app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -268,6 +268,18 @@ describe("the new project page", () => {
       .next()
       .should("have.text", "Not added");
 
+    cy.findByRole("button", { name: /submit/i }).should("be.disabled");
+    cy.get("body").happoScreenshot({
+      component: "Project Revision Summary",
+      variant: "no_change_reason",
+    });
+    cy.get("textarea").click().type("foo");
+    // Allow the component to finish saving before taking screenshot
+    cy.wait(1500);
```

Comment:
We usually try to avoid cy.wait() if possible. Could the same thing be achieved with something like this:
`cy.get("textarea").contains("foo");`
or `cy.contains("Changes saved").should("be.visible");` 
to be sure the changes have saved before taking the screenshot?
cy commands are also 'thenable' if that doesn't work